### DISCUSSION TITLE
[Scam Alert] Frz solar system (Frzss token)

### DISCUSSION THREAD

**Discussion by micro73** [Blockunity/WarningList#3] [General]
تحليل توكن   1401/04/03     frzss
ادرس کانترکت     0xaC41Fb8013c0B63588fC63997785A5d79E73eb28
ادرس سایت https://frzss.com/
ادرس ایمیل  info@frzss.com
Launched date: 02/22/2022
Total supp: 120,000,000,000,000
: $ 2,739,132 Diluted Market Cap
Price: 0.00000002242
 Volume 24h:  $217,448

 Cir supp : نامشخص 
١. با يك حساب سر انگشتي ماركت كپ تقسيم بر قيمت عدد بدست امده نشون ميده كه تمام توكن ها در گردش هست همچنین به دلیل نداشتن marketcap مشخص است که همه توکن ها در گردش است .

٢. 72 درصد توكن ها دست دو ولت است.(آپدیت جدید) همون ولت 18 درصد توکن سوزی کرد و الان 44 درصد توکن ها دست یک ولت است.

٣. تنها كميتوني كه دارد به شرح زير است: 
- تلگرام:  كانال زبان انگليسي 50 هزار دنبال کننده . بازدید کانال به شدت پایین و ادمین به هیچ عنوان جواب نمیدهد .
كانال فارسي زبان 54 هزار دنبال كننده كه ادمين جواب نميدهد و در هردو در حال توضيح ليستينگ پروژه و هيجانات كاذب به مخاطبين هستن همچنین بازدید از پست ها به شدت پایین نسبت به این تعداد فالوور
- توييتر 38 هزار دنبال كننده پست ها در مورد ليستينگ هست  و لایک و کامنت و تعامل به شدت پایین
- اينستاگرام ١ ميليون دنبال كننده كه فقط در مورد ليستينگ صحبت شده و بازدید و کامنت به شدت پایین .
* جايي نيست كه در مورد پروژه بتوان سوال كرد و جواب درست بگيري. 

٤.افراد پشت پروژه نامشخص و بدون هويت هستند. و خودشون را به هیچ عنوان معرفی نمیکنند  ولی طبق کانال تلگرام فارسی که دارند ایرانی هستند.

٥.مكان انجام پروژه يعني انرژي خورشيدي نامشخص هست و هيچ نشانه اي نيست

٦. در وايت پيپر پروژه اصلا در مورد خود پروژه صحبتي نشده و صحبت هاي خيلي سطحي مثل توضيح در مورد كاركرد شبكه بايننس اسمارت چين شده. 

٧. توكنوميك پروژه:  ( وایت پیپر پروژه اپدیت شده و این قسمت حذف شده است)
Dex: 2.08 %
Core team: 5% 
Partnership: 66.92% 
Airdrop: 1%
Developer: 15% 
مجموع اينها ميشه ٩٠ درصد 
پس ١٠ درصدش كو؟ 

٧. وايت پيپر پروژه خيلي سطحي همچنین از نظر فنی هیچ صحبتی نکردن که چگونه میخواهند این هدف(انرژی های سبز خورشیدی) را پیاده سازی کنند.
8. با بررسی در سایت bsc scan ولتی که الان 44 درصد توکن هارا دارد همان ولتی است که سازنده تمامی توکن هارا در ان دیپلوید کرده است.  
9. نمودار قیمتی degenerator  هست.
10. پروژه گیت هاب ندارد.
______________________________________________________________________________________________________________________________________

به تازگی گفتند که اسمارت کانترکت پروژه توسط contract wolf،بررسی و audit شده است ...
با یک سرچ ادرس سایت کانترکت ولف را پیدا کردم و با پشتیبانی تلگرام @ContractWolfGamma صحبت کردم و هزینه و تمامی موارد رو برای audit به شرح زیر ارسال کردند. قسمت های مختلف رو ترجمه میکنم و در اخر نتیجه گیری.
Gamma | Contractwolf.io
Good day!
services:
 A full contract audit would be only $149
•	بررسی کامل اسمارت کانترکت 149 دلار بیشتر هزینه ندارد.
Sample Report
(https://github.com/ContractWolf/smart-contract-audits/blob/main/ContractWolf_Audit_Sample.pdf)- (https://github.com/ContractWolf/smart-contract-audits/blob/main/ContractWolf_Audit_Infinity_Farms.pdf)
A full KYC/Doxxing with the project Owner or team member would be only $200
•	اگر اعضای تیم مشخص باشند با تایید هویتشان 200 دلار هزینه دارد .
Smart Contract Creation/ Development
FREE ADVERTISEMENT post on :
Twitter (https://twitter.com/contractwolf_io) (29.6k Followers)
Telegram Channel (https://t.me/ContractWolfOfficial) (18.8k followers)
Our private channel has a lot of potential participants and investors for your project
•	اگر قراردادهوشمند رو به این شرکت بدهید در عوض تبلیغات رایگان هم برایتان انجام میدهند در توییتر و تلگرام خودشون
A quality contract testing from our teams software engineers
consisting of Manual Audit, Remix, Ganache, Open Zeppelin and Truffle test and SWC(Smart Contract Weakness Classification) for our Penetration/security tests
Standard contract audits can be delivered within 8-12 hours
For rush orders 1-2 hours turnaround with an additional payment of $100
•	*تقریبا بررسی هر اسمارت کانترکت 8 تا 12 ساعت طول میکشد اگه میخواهید سفارش سریعتر باشد باید 100 دلار بیشتر بپردازید .
Check us at :
WEBSITE
(https://contractwolf.io/)
 TWITTER
(https://twitter.com/contractwolf_io)
TELEGRAM
(https://t.me/ContractWolfOfficial)
GITHUB
(https://github.com/ContractWolf/smart-contract-audits)
•	نتیجه اینکه حدودا با مبلغ 160 دلار یک اسمارت کانترکت قابل بررسی می باشد و اینکه بررسی یا audit یک پروژه به تنهایی مورد خوبی برای تصمیم به سرمایه گذاری نیست و تمامی موارد باید با دقت بررسی شوند
______________________________________________________________________________________________________________________________________

_### شرکت contractwolf در ابتدای پی دی اف بررسی و audit توکن frzss هشدار هایی نوشته که به صورت عمومی برای تمامی پروژه های بلاکچینی صادق است_

سلب مسئولیت: 
حسابرسی ها و گزارش های ContractWolf.io نباید به عنوان شکلی از "تبلیغات" پروژه در نظر گرفته شود و هیچ گونه تعامل و ارزیابی از "قرارداد پروژه" تا "قراردادهای خارجی" مانند Pancakeswap یا موارد مشابه را پوشش نمی دهد.
ContractWolf
 هیچ ضمانتی در مورد گزارش های منتشر شده خود ارائه نمی دهد.
ContractWolf 
نباید به عنوان تصمیمی برای سرمایه گذاری در یک پروژه حسابرسی شده استفاده شود و به پروژه های قراردادی حسابرسی شده آن وابسته یا شریک نیست.
ContractWolf 
گزارش شفافی را برای همه "مشتریان" خود و "شرکت کنندگان مشتریان" خود ارائه می دهد و هیچ تضمینی برای کدهای بدون اشکال در قرارداد SMART خود نخواهد داشت.
حضور ContractWolf برای تجزیه و تحلیل، ممیزی و ارزیابی کد قرارداد هوشمند مشتری است.
هر شرکت یا پروژه باید در قبال نقص ها و عملکردهای امنیتی خود پاسخگو باشد.
باشد که بیندیشید و بررسی کنید و بعد سرمایه گذاری کنید.
_____________________________________________________________________________________________________________________________________

در اخرین اقدام انجام شده توسط مالک توکن در کانال تلگرامی فارسی پروژه در حال پامپ و دامپ قیمت توسط خودش هست .
به صورت شفاف خودش بیان میکند در حال خرید و پامپ قیمت هستم و اسناد وتصاویر خرید در کانالش موجود هست

در حال حاضر به جای صحبت در مورد کاربرد پروژه و پیشرفت پروژه فقط در حال پامپ و دامپ قیمت هستند
به کاربران و خریداران نااگاه هیجان میدهند
•	بخرید و نفروشید تا ترند شویم
•	نوسانگیر ها خارج شوند !
•	اسم ما همه جا باید باشه
اما دریغ از صحبت در مورد کارکرد پروژه !!!!
______________________________________________________________________________________________________________________________________

گزارش tokensniffer برای این توکن
https://tokensniffer.com/token/g9l5bs6izjuvx60bxjkm01pvvzdb5k0chhqqv41941f2qlh3gauudqu0ajnv

bubble map نقشه حبابی ادرس های کیف پول

https://tokensniffer.com/bubble/bsc/0xac41fb8013c0b63588fc63997785a5d79e73eb28
طبق گفته سایت توکن اسنیفر
حباب های ابی رنگ هولدر های معمولی هستند ولی نباید بسیار بزرکتر از بقیه باشد که در اینجا هست
حباب های نارنجی رنگ ولت سازدنگان توکن هست دو ولت ساذنده هست یکی از ان خیلی زیاد توکن دارد
حباب های قرمز رنگ کیف پول burning هست طبق این سایت کیف پول سوزاندن ندارد
_____________________________________________________________________________________________________________________________________

به تازگی سورس کد این توکن توسط certik هم بررسی شده که روی ان خیلی مانور میدهند 
باید توجه داشت که بررسی سورس کد یکی از چندین مواردی است که باید در تحقیقات بررسی شود و دلیل خوب بودن این توکن نمی باشد.

Disclaimer:CertiK conducts security assessments on the provided source code exclusively. Conduct your own due diligence before deciding to use any info listed at this page.
جمله استفاده شده در بررسی سورس کد تمامی پروژه های بررسی شده توسط certik
سلب مسئولیت: CertiK ارزیابی های امنیتی را منحصراً روی کد منبع ارائه شده انجام می دهد. قبل از تصمیم به استفاده از اطلاعات فهرست شده در این صفحه، بررسی های لازم را انجام دهید.

**Comment by micro73** [comment]
طبق audit سورس کد های توکن توسط certik، سورس کد این پروژه دارای ایراداتی به شرح زیر است :

- ۲ تا ایراد عمده داره
که میشه توزیع و مخفی بودن اطلاعات

- ۳ تا ایراد نرمال داره که شامل
ارور کامپایلر
ساده بودن کد ها از نظر امنیت
نوع نوشتار کد و ایرادات اساسی در کد

- ۳ تا ایراد جزئی داره
استفاده نکردن از متغییر برای گس فی
نوشتار کد ها ضعیف و کد های اضافی
تکرار زیاد کد ها

- ۳ تا ایراد اطلاعاتی داره
نسخه کامپایلر انلاک شده 
پیام های خطا دار حذف شدند
ادرس صفر و اعتبار سنج نداره

- نمره خیلی کمی گرفته
با توجه به ۱۱ تا ایراد امتیاز زیر ۷۰ میگیره
ولی اجازه شیر اطلاعات نداده و فقط ایرادات رو نشون میده

**Comment by Blockunity** [comment]
این پروژه اسکم شد.
![IMG_20230217_193924_105.jpg](https://user-images.githubusercontent.com/72666334/219756822-33a3add8-a317-4a51-bb1e-06917917454d.jpg)
